mteb/abstasks/AbsTaskClusteringFast.py

```diff
@@ -125,11 +125,11 @@ def evaluate(self, model, split="test", **kwargs) -> Scores | Dict[HFLang, Score
                 logger.info(
                     f"\nTask: {self.metadata.name}, split: {split}, language: {lang}. Running..."
                 )
-                _ds = multilingual_ds[lang]
                 multilingual_scores[lang] = self._evaluate_monolingual(
                     model, multilingual_ds[lang], split, **kwargs
                 )
-                return multilingual_scores
+                self._add_main_score(multilingual_scores[lang])
```

Comment:
Not sure if we need this here since it's only using v-measure. I copied this structure from `AbsTaskClustering`.